tools/scripts/xilinx.mk

```diff
@@ -2,24 +2,22 @@
 #                           ENVIRONMENT VARIABLES                              
 #------------------------------------------------------------------------------
 
+# Used by the build utils
+export HARDWARE
+export WORKSPACE
+export TEMP_DIR
```

Comment:
Can the variable been added as parameters instead of exported?
Will be more issue proof.

Reply:
As workspace, tmp dir, etc are always the same we can avoid over parametrising everything.